android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt

```diff
@@ -187,7 +215,7 @@ class OpenloginFlutterPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, P
 
     return LoginParams(getOpenLoginProvider(provider),
             appState = appState,
-            reLogin = reLogin,
+            relogin = reLogin,
```

Comment:
pls make the user exposed sdk same as web. meaning change case of relogin

Reply:
done